File: src/main/java/uk/co/threebugs/mochiwhattotrade3/MetaTraderDir.java
package uk.co.threebugs.mochiwhattotrade3;

import com.fasterxml.jackson.annotation.JsonProperty;

public class MetaTraderDir {
    @JsonProperty("name")
    private String name;

    @JsonProperty("dir-path")
    private String dirPath;

    // Getters and setters
    public String getName() {
        return name;
    }

    public String getDirPath() {
        return dirPath;
    }
}

File: src/main/kotlin/uk/co/threebugs/mochiwhattotrade3/MetaTraderDir.kt
package uk.co.threebugs.mochiwhattotrade3

import com.fasterxml.jackson.annotation.JsonProperty

class MetaTraderDir {
    // Getters and setters
    @JsonProperty("name")
    val name: String? = null

    @JsonProperty("dir-path")
    val dirPath: String? = null
}
